### DISCUSSION TITLE
Are buttons disabled?

### DISCUSSION THREAD

**Discussion by Wolfhound905** [kiki7000/discord.py-components#4] [Q&A]
Are buttons working anymore?

**Comment by kiki7000** [comment]
Discord has made buttons and dropdowns available only to beta users, so buttons can't be created, used by normal users right now.
However, you can use the button and this module normally when it is released.

**Comment by aeongdesu** [comment]
Now available~
[API documentation](https://discord.com/developers/docs/interactions/message-components)
